src/main/java/com/zufar/onlinestore/payment/dto/PriceDetailsDto.java

```diff
@@ -9,7 +9,7 @@
 public record PriceDetailsDto(
 
         @NotNull(message = "Total price is mandatory attribute")
```

Comment:
update the message with the new field name, please
`@NotNull(message = "ItemsTotalPrice is mandatory attribute")`